### ISSUE TITLE
Option to show/control timezone in ikhal

### ISSUE THREAD

**Issue by madduck** [pimutils/khal#761]
For those of us who travel timezones all the time, it'd be great to be able to configrue `ikhal` to display the current timezone in the top right corner, and also display timezones next to all times of events.

Finally, if I could change the current timezone within ikhal, that would rock. I can of course restart it with a different value in the `TZ` environment variable, but if I just want to quickly switch the view because I'm e.g. on the phone trying to arrange for a meeting next week (in another timezone), having this functionality would be awesome!

**Comment by geier** [comment]
Agreed, this is on the todo list, but is rather difficult to do (or better: it needs a lof of test, because changing time zones wrongly may lead to unintended consequences).

**Comment by varac** [comment]
I'm working in a distributed team across multiple timezones and multiple daylight-saving changes makes it a great mess when you lack proper timezone  support (I'd like to schedule some meetings in based on UTC i.e.).
Please implement this soon, this is the only major feature missing for me.
